spec/features/devise/confirmations_new_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.feature 'Resend confirmation instructions', type: :feature do
+  scenario 'User requests resend with unconfirmed email' do
+    # Create a user with unconfirmed email
+    create(:user, email: 'user@example.com', password: 'password', confirmed_at: nil)
+
+    visit new_user_confirmation_path
+
+    within('.signup-container') do
+      expect(page).to have_field('Email', type: 'email')
+      expect(page).to have_selector('input[type="submit"].signup-submit', visible: :hidden)
+    end
+
+    fill_in 'Email', with: 'user@example.com'
+    execute_script("document.getElementById('confirmation-form').submit();")
```

Comment:
- [ ] The below tests are failing, Please make sure that all tests pass without any errors.

![image](https://github.com/ometman/budget-app/assets/83803414/882219a5-3eb3-4b15-84c4-ff62c9dde487)

Reply:
Thanks. It's fixed. All tests now pass successfully.